docx/docx.go

```diff
@@ -31,13 +31,59 @@ func (config *Config) BuildDocx() error {
 	funcMap := template.FuncMap{
 		"getAllControlSections": openControl.FormatControl,
 		"getControlSection":     openControl.FormatControlSection,
+		"getParameter":          openControl.FormatParameter,
+		"getResponsibleRole":    openControl.FormatResponsibleRoles,
 	}
 	docTemplate.AddFunctions(funcMap)
 	docTemplate.Parse()
 	docTemplate.Execute(config.ExportPath, nil)
 	return err
 }
 
+// FormatResponsibleRole fills in the responsible role for each component for a given standard and control.
+func (openControl *OpenControlDocx) FormatResponsibleRoles(standardKey string, controlKey string) string {
+	var text string
+	openControl.Justifications.GetAndApply(standardKey, controlKey, func(selectJustifications models.Verifications) {
+		for _, justification := range selectJustifications {
+			openControl.Components.GetAndApply(justification.ComponentKey, func(component *models.Component) {
+				// Print out the component name and the responsible for that component.
+				if justification.SatisfiesData.ResponsibleRole != "" {
+					text = fmt.Sprintf("%s%s: %s\n", text, component.Name, justification.SatisfiesData.ResponsibleRole)
+				}
+			})
+		}
+	})
+	return text
+}
+
+// FormatParameter fills in the parameter for a given parameter, standard and control.
+func (openControl *OpenControlDocx) FormatParameter(standardKey string, controlKey string, parameterKey string) string {
+	var text string
+	openControl.Justifications.GetAndApply(standardKey, controlKey, func(selectJustifications models.Verifications) {
+		for _, justification := range selectJustifications {
+			openControl.Components.GetAndApply(justification.ComponentKey, func(component *models.Component) {
+
+				// Look if there's parameter with the same key as the passed in parameterKey.
+				found := false
+				var parameterText string
+				for _, parameter := range justification.SatisfiesData.Parameters {
+					// If section header exists, let's print it's corresponding text and not the header itself.
+					if parameter.Key == parameterKey {
+						parameterText = fmt.Sprintf("%s%s\n", parameterText, parameter.Text)
+						found = true
+					}
+				}
+				// If we actually found a section, let's prepend the component name to the parameter.
+				if found {
+					text = fmt.Sprintf("%s%s:\n%s", text, component.Name, parameterText)
+				}
```

Comment:
Ditto the question in #143 about what the behavior for `found == false` should be.